moby/swarmkit
# Add OOM adjustment flag to hostconfig via swarmctl cli

**- What I did**
Add kernel level tuning flag for OOM adjustment to hostconfig via swarmctl cli

**- How I did it**
I added flags to swarmctl modified the API to include oom-score-adj flag.

**- How to test it**

After starting swarmd, run

swarmctl service create --name <service_name> --oom-score-adj <score> --image <image_name>

Verify oom-score-adj works correctly by inspecting the container

docker container inspect <container_name>

The Hostconfig, OomScoreAdj value should be set to the value you specified via swarmctl


**- Description for the changelog**
Add OOM adjustment flag to hostconfig via swarmctl cli
